tests/test_arraycreation.py

```diff
@@ -716,3 +701,85 @@ def test_set_shape(shape):
     da.shape = shape
 
     assert_array_equal(na, da)
+
```

Comment:
115 tests are failed in the internal CI when running on Iris XE. The errors have to be addressed and resolved.